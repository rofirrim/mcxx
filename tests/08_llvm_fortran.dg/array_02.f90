! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_02.f90.out"
! </testinfo>
PROGRAM MAIN
    REAL :: A(3)

    A(1) = 1.2
    A(2) = 2.3
    A(3) = 3.4

    PRINT *, A

END PROGRAM MAIN
