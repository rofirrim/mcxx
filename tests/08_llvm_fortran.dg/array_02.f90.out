   1.20000005       2.29999995       3.40000010    
